force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -139,40 +203,40 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static String returnHelloWorld() {
         // Initialize the result String
-
+        String result = '';
         // Use a for loop to append 'Hello World!;' to the result string three times
-        /* 
-         for (){
-            
+        for (Integer i=0; i<3; i++){
+            result = result +'Hello World!; ' ;
+            system.debug(result);
          }
-        */
-
-        return null; // Replace null with the variable you used to store the result
+        return result; // Replace null with the variable you used to store the result
     }
 
     /**
      * Question 10
      * Generate a string where the input string is repeated the given number of times, with each
-	 * repetition separated by a semicolon. 
+* repetition separated by a semicolon.
      * Do not add a semicolon after the last repetition.
-	 * Example: repeatString("Hello World!", 3) should return "Hello World!; Hello World!; Hello World!"
+* Example: repeatString("Hello World!", 3) should return "Hello World!; Hello World!; Hello World!"
      * @param inputString The string to be repeated.
      * @param repeatCount The number of times the string should be repeated.
      * @return The new string with the inputString repeated repeatCount times.
      */
     public static String repeatString(String inputString, Integer repeatCount) {
         // Initialize the result String
-
+        String result = '';
         // Use a for loop to append the inputString to the result string repeatCount times
-        /*
-        for () {
-
+        for (Integer i=0; i<repeatCount; i++) {
+            result = result+ inputString;
+            system.debug(result);
             // If it is not the last iteration, add a semicolon to separate the strings
+            if(i<repeatCount-1){
+                result = result + '; ';
+                system.debug(result);
+            }
         }
-        */
-
         // Return the final result string
-        return null; // Replace null with the variable you used to store the result
+        return result; // Replace null with the variable you used to store the result
     }
```

Comment:
Good job. Take a look at the String methods repeat and removeEnd. 

https://developer.salesforce.com/docs/atlas.en-us.apexref.meta/apexref/apex_methods_system_string.htm